### PR TITLE
Fixed "Background" is misspelled, bug

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey ! I have fixed this. Please review my PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
   <head>
     <title>Scratch Empires Home</title>
   </head>
-  <body Backgorund-image: url("https://raw.githubusercontent.com/Autofirejm/Scratch-Empires/main/2022-01-20_13.43.01.png")>
-    <H1 style="text-align:center; color:"#ffffff""> Scratch Empires Website</H1>
+  <body backgorund-image: url("https://raw.githubusercontent.com/Autofirejm/Scratch-Empires/main/2022-01-20_13.43.01.png")>
```
Comment (ghost) [inline index.html]: Still misspelled...

Comment (NaincyKumariKnoldus) [reply index.html]: Hey @ILOVETOMAKESTUFF I have fixed now again.

**Comment by Autofirejm** [comment]
Whoops! forgot to close this...

**Comment by NaincyKumariKnoldus** [comment]
Hey @Autofirejm You have not merged my PR.

**Pull request closed without merging**